### ISSUE TITLE
Portable mode where user data is saved in a folder together with the program

### ISSUE THREAD

**Issue by ww-9** [ransome1/sleek#557]
Portable mode request for v2.0.

On Windows userData folder is saved in user's AppData folder in the new version. In the previous user-preferences.json was saved in the program folder.  Request to implement a portable mode where all settings are saved in a folder together with the program.

It would be even better if the relative paths to the .txt files are saved, so that they can be saved there in the folder.

**Comment by ransome1** [comment]
Saving the user preferences in the same folder than the application will require to save the userData folder in that folder. That's because in v2.x we now have 2 configuration files, one for settings and one for persisted filters. And there can be potentially more of course, if future feature require this. Is this what is expected from a portable Windows app? I don't have a lot of experiences with Windows. 

The relative file paths might be a bit more tricky and need a significant amount of research prior to implementation. 

I would suggest to narrow down the scope here to the main issue. Otherwise we will be dragging this issue with us for a long time. A separate feature request is a good idea. Please make it as detailed as possible.

**Comment by ransome1** [comment]
I fully agree. It would only be the actual 2 configuration files. Still unsure if just as it or in a folder, just to have both nearly aggregated.

**Comment by ww-9** [comment]
Definitely in a folder, if something else has to be added in the future

**Comment by ransome1** [comment]
@ww-9 @AaronKnowls @GL-Doses this behavior should now work with the portable version of the latest pre-release: https://github.com/ransome1/sleek/releases/tag/v2.0.28-rc.1
